test/Java8andUp/src/org/openj9/test/vmArguments/VmArgumentTests.java

```diff
@@ -304,12 +312,10 @@ public void testArgEncodingInIbmJavaOptions() {
 		String ibmJavaOptionsArg = "-Xargencoding";
 		env.put(IBM_JAVA_OPTIONS, ibmJavaOptionsArg);
 		ArrayList<String> actualArguments = runAndGetArgumentList(pb);
-		HashMap<String, Integer> posns = checkArguments(actualArguments, new String[] {ibmJavaOptionsArg});
-		assertTrue("missing argument: "+ibmJavaOptionsArg, posns.containsKey(ibmJavaOptionsArg));
-		Integer ibmOptionsPosn = posns.get(ibmJavaOptionsArg);
+		HashMap<String, Integer> argumentPositions = checkArguments(actualArguments, new String[] {ibmJavaOptionsArg});
+		assertTrue("missing argument: "+ibmJavaOptionsArg, argumentPositions.containsKey(ibmJavaOptionsArg));
 		/* environment variables should come after implicit arguments */
-		int extDirsPosn = posns.get(DJAVA_HOME);
-		assertTrue(IBM_JAVA_OPTIONS+ " should come last", ibmOptionsPosn > extDirsPosn); /* this will fail on current VMs */
+		assertTrue(IBM_JAVA_OPTIONS+ " should come last", argumentPositions.get(ibmJavaOptionsArg).intValue() > argumentPositions.get(DJAVA_HOME).intValue());
 		}
```

Comment:
Since this function is being changed anyway, can you fix the indenting on this `}`?

Reply:
Done.